### DISCUSSION TITLE
Welcome to awesome-graphql-client Discussions!

### DISCUSSION THREAD

**Discussion by lynxtaa** [lynxtaa/awesome-graphql-client#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by stasdavydov** [comment]
Thank you for the awesome-graphql-client library!
